kubernetes.yaml

```diff
@@ -0,0 +1,153 @@
+package:
+  name: kubernetes
+  version: 1.27.3
+  epoch: 1
+  description: Production-Grade Container Scheduling and Management
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - bash
+      - ca-certificates-bundle
+      - build-base
+      - go
+      - go-bindata
+      - linux-headers
+      - grep
+      - jq
+      - coreutils # needed for non busybox version of `mktemp`
+      - findutils # needed for non busybox version of `xargs`
+      - rsync
+      - apk-tools
+
+# "transform" the kubernetes version into the corresponding pause version, these don't always line up
+var-transforms:
+  - from: ${{package.version}}
+    match: ^(.+)$
+    replace: 3.9
+    to: pause-version
+
+vars:
+  components: "kubectl kubeadm kubelet kube-scheduler kube-proxy kube-controller-manager kube-apiserver"
```

Comment:
Do we want this to be a `range` with subpackages that `make` each one, etc.? Or is the idea that this gives us a `kubernetes` package with allll the things, and subpackages for each individual thing if we want that.

Either way, +1 to this change.

Reply:
yeah +1 to your point. I started down the `range` route then fell down an overengeered templating rabbit hole.

I think while this is more verbose, it's easier to understand at first glance, and easier to maintain long term.